bfv/operand.go

```diff
@@ -82,23 +82,6 @@ func (el *Element) Degree() uint64 {
 	return uint64(len(el.value) - 1)
 }
 
-// Resize resizes the target Element degree to the degree given as input. If the input degree is bigger, then
-// it will append new empty polynomials; if the degree is smaller, it will delete polynomials until the degree matches
-// the input degree.
-func (el *Element) Resize(params *Parameters, degree uint64) {
```

Comment:
This was a public function. Is there a specific reason for removing it beside that its not used/tested ?

Reply:
Following the code review suggestion, unused code should be removed. As far as I am concerned, this function has never been used and has no real purpose anymore since the evaluator now enforces the output ciphertext to be of the correct size.

Reply:
It might be used in user code, since `Resize` is an exported function of e.g., `Ciphertext`.

If we remove it, we need to have a line of the changelog about that.

Reply:
We will keep it, to make it symmetric with CKKS. In addition, the .Level() method has been added and might be needed in the future.